kDrive/UI/Controller/Files/FileActionsFloatingPanelViewController+Actions.swift

```diff
@@ -72,8 +72,10 @@ extension FileActionsFloatingPanelViewController {
                 return file.capabilities.canBecomeDropbox
             case .manageDropbox:
                 return file.isDropbox
+            case .upsaleColor:
+                return file.isDirectory && driveFileManager.drive.isFreePack
             case .folderColor:
-                return file.capabilities.canColor
+                return file.capabilities.canColor && !driveFileManager.drive.isFreePack
```

Comment:
Something I dislike here is that without a check for the free pack, I got a second identical action displayed if we rely only on the `canColor` of the backend.